_build/requirements.yml

```diff
@@ -7,7 +7,9 @@ collections:
   - name: google.cloud
   - name: openstack.cloud
   - name: community.vmware
-  - name: ovirt.ovirt
+  - name: https://github.com/shanemcd/ovirt-ansible-collection.git
```

Comment:
Is it okay that referring collections from personal repo?

Reply:
The patch in the private fork is now in master at https://github.com/oVirt/ovirt-ansible-collection.git, so could use that instead.